scripts/docker-integration-tests/prometheus_replication/test.sh

```diff
@@ -14,14 +14,12 @@ echo "Pull containers required for test"
 docker pull $PROMREMOTECLI_IMAGE
 docker pull $JQ_IMAGE
 
-echo "Run m3dbnode and m3coordinator containers"
-docker-compose -f ${COMPOSE_FILE} up -d dbnode01
-docker-compose -f ${COMPOSE_FILE} up -d dbnode02
-docker-compose -f ${COMPOSE_FILE} up -d coordinator01
-docker-compose -f ${COMPOSE_FILE} up -d coordinator02
+echo "Run etcd, m3dbnode and m3coordinator containers"
+docker-compose -f ${COMPOSE_FILE} up -d
 
 function defer {
-  docker-compose -f ${COMPOSE_FILE} down || echo "unable to shutdown containers" # CI fails to stop all containers sometimes
+  :
+#  docker-compose -f ${COMPOSE_FILE} down || echo "unable to shutdown containers" # CI fails to stop all containers sometimes
```

Comment:
Do we not need this anymore? If not, why not just remove the entire function the the `trap defer EXIT` below?